dashboard/lib/tasks/seed.rake

```diff
@@ -185,8 +185,11 @@ namespace :seed do
     script_files = opts[:script_files] || SCRIPTS_GLOB
     begin
       custom_scripts = script_files.select {|script| File.mtime(script) > scripts_seeded_mtime}
-      _, custom_i18n = Script.setup(custom_scripts, show_progress: Rake.application.options.trace)
-      Script.merge_and_write_i18n(custom_i18n)
```

Comment:
I think it's safe to ignore i18n here, because the only time any i18n was getting generated was when ScriptDSL found some in the .script file here: https://github.com/code-dot-org/code-dot-org/blob/28cc92d20cb5ef14547153327a1d006c80c19e0f/dashboard/app/models/script.rb#L1047-L1049

this begs the question, _should_ seeding migrated units update i18n? I think the answer is no, because seeding doesn't seem like the right time to update i18n. I would expect that only to be necessary when a script is updated on levelbuilder via LessonsController#update. 

a practical argument for letting this behavior go away is that this takes us closer to the levelbuilder machine owning scripts.en.yml, which would reduce the chance of merge conflicts on scripts.en.yml. the old code allowed every environment to merge i18n into scripts.en.yml during seeding, which has resulted in failed deploys not just in staging but also test and production.

If we're comfortable with this path, we should inform other engineers and let them know to use levelbuilder mode locally if they want to edit units locally from now on. then the unit update codepath will take care of updating scripts.en.yml.